front-end/src/Scenes/LogIn/Containers/logIn.css

```diff
@@ -0,0 +1,27 @@
+.layout{
+    height: 100vh;
+    background: #fcfcfc;
+    color: #fff;
+    display: flex;
+}
+.header {
+    height: 4vw;
```

Comment:
Minor: still unexpected values, please do not overuse viewport dimensions
Use px and %, 
vh, vw is for an exceptional situation.